mountpoint-s3-fs/src/manifest/builder.rs

```diff
@@ -1,62 +1,392 @@
-use super::ManifestError;
+use std::collections::HashSet;
+use std::io::BufReader;
+use std::path::{Path, PathBuf};
+use std::{collections::HashMap, fs::File};
+
+use fuser::FUSE_ROOT_ID;
```

Comment:
nit: we already import this constant somewhere (metablock?). Let's use it from there.